### DISCUSSION TITLE
Proposal : Languages tab in reatom.dev

### DISCUSSION THREAD

**Discussion by caio2983** [reatom/reatom#976] [Ideas]
A language switching tab in the docs website would be useful for supporting other languages besides english . This could expand the amount of people understanding and using Reatom

What do you guys think ?
I could translate the docs entirelly to PT-BR . People who know other languages could contribute to this idea too

**Comment by artalar** [comment]
It is a nice idea. The original community mostly knows Russian, and people would love to see the content in the original language and contribute to it. However, I'm not sure if it is a reasonable effort given the potential workload. We already have a lot of documentation, and this is just the beginning — I plan to create many small guides in the coming year. How will we sync changes across languages? How will we handle differences between translations? Should we accept additional documentation in languages other than the base (English) language? There are many management challenges that concern me.

**Reply by caio2983** [reply]
I see, this could be an unexpectedly big task
